lib/tests.sh

```diff
@@ -223,28 +223,28 @@ _VALIDATE_THAT_APP_CAN_BE_ACCESSED () {
             [ $curl_error -eq 1 ] && log_error "The HTTP code shows an error."
         fi
 
-        # Analyze the output of curl
+        # Analyze the output of cURL
         if [ -e "$curl_output" ]
         then
             # Print the title of the page
             local page_title=$(grep "<title>" "$curl_output" | cut --delimiter='>' --fields=2 | cut --delimiter='<' --fields=1)
             local page_extract=$(lynx -dump -force_html "$curl_output" | head --lines 20 | tee -a "$complete_log")
 
-            # Check if the page title is neither the YunoHost portail or default nginx page
+            # Check if the page title is neither the YunoHost portail or default NGINX page
             if [ "$page_title" = "YunoHost Portal" ]
             then
                 log_debug "The connection attempt fall on the YunoHost portal."
                 fell_on_sso_portal=1
-                # Falling on nginx default page is an error.
-            elif echo "$page_title" | grep -q "Welcome to nginx"
+                # Falling on NGINX default page is an error.
+            elif echo "$page_title" | grep -q "Welcome to NGINX"
```

Comment:
Eeeh is it this one really a typo ? I'm thinking it's probably related to the actual content of the default nginx page ?

Reply:
You are right! Silly me. Thanks for the checking

Reply:
Fixed at https://github.com/YunoHost/package_check/pull/110